README.md

```diff
@@ -122,7 +124,24 @@ Then run `test/run_tests.sh` and `test/cpp/run_tests.sh` to verify the setup is
 
 
 
-# How To Install Pre Built PyTorch TPU Wheels
+# Using Pre Built Releases 
+
+## Pre Built Docker Images (recommended)
+
+Docker images with `torch` and `torch_xla` preinstalled in the `pytorch` conda
+environment are distributed under: `gcr.io/tpu-pytorch/xla`. This image has two type of tags which take the forms of:
+
+* gcr.io/tpu-pytorch/xla:nightly
+* gcr.io/tpu-pytorch/xla:YYYYMMDD (ex. gcr.io/tpu-pytorch/xla:nightly_20190531)
+
+With these images, for example, you can train mnist on TPUs with the following commands:
+
+```Shell
+docker pull gcr.io/tpu-pytorch/xla:nightly
```

Comment:
I would also add an line that shows how to run a shell within the Docker.
Many knows, but people that are new to it, they might be inclined to think that they need to run the docker every time.

Reply:
Good idea! Done.